api/src/controllers/collections.ts

```diff
@@ -101,6 +137,37 @@ router.patch(
 	respond
 );
 
+router.post(
+	'/:collection',
+	multipartHandler,
```

Comment:
This shouldn't be under `/collections/:collection`, as per the previous comment

Reply:
So which route you will suggest? Could it be /collections/:collection/import?

Reply:
`/collections` is used for schema management, so that's out of the question to me. I'm thinking either `POST /items/:collection` with a file, or `POST /items/:collection/import`

Reply:
Ok, will fix it